ssl/ssl_conf.c

```diff
@@ -520,6 +520,22 @@ static int cmd_DHParameters(SSL_CONF_CTX *cctx, const char *value)
     return rv > 0;
 }
 #endif
+
+static int cmd_RecordPadding(SSL_CONF_CTX *cctx, const char *value)
+{
+    size_t block_size = (size_t)atoi(value);
```

Comment:
The SSL[_CTX]_set_block_padding() functions check the limits, so all we need to do here is make sure it's positive.